pages/linux/unshadow.md

```diff
@@ -0,0 +1,12 @@
+# unshadow
+
+> Utility provided by the John the Ripper project to obtain the traditional Unix password file if the system uses shadow passwords.
+> More information: <https://www.openwall.com/john/>.
+
+- Combine the /etc/shadow and /etc/passwd of the current system:
```

Comment:
I think these 2 examples should be just one. We always use tokens for options in commands. The value inside them can be concrete or generic. I had made the same comment some time earlier too. This is what I'd suggest:

Combine any two shadow and password files:

`sudo unshadow {{/etc/passwd}} {{/etc/shadow}}`.